filament/src/Froxelizer.cpp

```diff
@@ -197,8 +197,8 @@ bool Froxelizer::prepare(
     assert(mFroxelShardedData.begin());
 
 #ifndef NDEBUG
-    memset(mFroxelBufferUser.data(),    0x55, mFroxelBufferUser.sizeInBytes());
-    memset(mRecordBufferUser.data(),    0xEB, mRecordBufferUser.sizeInBytes());
+    memset(mFroxelBufferUser.data(), 0x55, mFroxelBufferUser.sizeInBytes());
```

Comment:
Why this change?

Reply:
That's strange, I don't recall modifying this 👎